sentry_telegram/plugin.py

```diff
@@ -124,6 +138,12 @@ def send_message(self, url, payload, receiver):
         )
         self.logger.debug('Response code: %s, content: %s' % (response.status_code, response.content))
 
+    def send_message_through_proxy(self, url, payload, receiver, proxy):
```

Comment:
This method looks like almost copy of `send_message` method.
I propose to use one `send_message`, and add proxies like this:
```
response = safe_urlopen(
    method='POST',
    url=url,
    json=payload,
    params=self.get_request_params(),
)
```
Where `get_request_params()` will return proxies settings, if it required.